src/js/providers/html5.js

```diff
@@ -520,6 +557,7 @@ define([
             _source = _levels[_currentQuality];
             _position = item.starttime || 0;
             _duration = item.duration || 0;
+            _itemTracks = item.tracks;
```

Comment:
I actually had it that way for that reason using ```arguments```. I agree with storing ```_item``` if we need to, but at this point, we don't. I'll remove ```_itemTracks```.